src/app/pages/datasets/components/dataset-details-panel/dataset-details-panel.component.html

```diff
@@ -1,7 +1,8 @@
-<h3>Details for {{ dataset.name }}</h3>
+<h3 class="title">Details for {{ dataset.name }}</h3>
 
 <div class="cards">
   <ix-permissions-card [dataset]="dataset"></ix-permissions-card>
+  <ix-dataset-details-card *ngIf="dataset.id !== dataset.pool" [dataset]="dataset"></ix-dataset-details-card>
```

Comment:
This condition should be removed. The root dataset has the same name as the pool and has details we need to show